test/src/system/st_3drepobouncerClient.cpp

```diff
@@ -104,11 +104,12 @@ static std::string produceUploadArgs(
 	const std::string &filePath,
 	const std::string &configPath = getConnConfig())
 {
-	return  getClientExePath()
+	std::string path = getClientExePath()
```

Comment:
why do you need this change?

Reply:
This change was done for debugging purposes. Removing this unnecessary path variable.